packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -18,28 +17,26 @@ const selectors = {
 /**
  * Component representing the sidebar on the dashboard of WPCOM.
  *
- * @augments {BaseContainer}
  */
-export class SidebarComponent extends BaseContainer {
-	sidebar!: ElementHandle;
+export class SidebarComponent {
+	private page: Page;
 
 	/**
-	 * Constructs an instance of the component.
+	 * Get the sidebar ElementHandle
 	 *
-	 * @param {Page} page The underlying page.
+	 * @returns the ElementHandle for the sidebar
 	 */
-	constructor( page: Page ) {
-		super( page, selectors.sidebar );
+	private async selectSidebar(): Promise< ElementHandle > {
+		return await this.page.waitForSelector( selectors.sidebar );
 	}
 
 	/**
-	 * Post-initialization steps of this object.
+	 * Constructs an instance of the component.
 	 *
-	 * @returns {Promise<void>} No return value.
+	 * @param {Page} page The underlying page.
 	 */
-	async _postInit(): Promise< void > {
-		await this.page.waitForLoadState( 'domcontentloaded' );
-		this.sidebar = await this.page.waitForSelector( selectors.sidebar );
+	constructor( page: Page ) {
+		this.page = page;
```

Comment:
To clarify; I assume we would need to rebase any in-progress branches once this lands on trunk?

Reply:
Yep! Anything that is calling `SidebarComponent.Expect` would need to be updated to just use the constructor. But everything after that should work just the same.